jgrapht-core/src/main/java/org/jgrapht/traverse/ClosestFirstIterator.java

```diff
@@ -55,7 +55,9 @@
     private boolean initialized = false;
 
     /**
-     * Creates a new closest-first iterator for the specified graph.
+     * Creates a new closest-first iterator for the specified graph.  Iteration
```

Comment:
Would it make sense to deprecate this one, and force users to always provide at least one start vertex?

Reply:
Yep, will do.